aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -35,24 +46,36 @@ module aptos_framework::staking_config {
         allow_validator_set_change: bool,
         // The maximum rewards given out every epoch. This will be divided by the rewards rate denominator.
         // For example, 0.001% (0.00001) can be represented as 10 / 1000000.
-        rewards_rate: u64,
-        rewards_rate_denominator: u64,
+        _rewards_rate: u64,
+        _rewards_rate_denominator: u64,
         // Only this % of current total voting power is allowed to join the validator set in each epoch.
         // This is necessary to prevent a massive amount of new stake from joining that can potentially take down the
         // network if corresponding validators are not ready to participate in consensus in time.
         // This value is within (0, 50%), not inclusive.
         voting_power_increase_limit: u64,
     }
 
+    struct StakingRewardsConfig has copy, drop, key {
+        yearly_rewards_rate: u64,
```

Comment:
Why not directly store per second reward rate?

Reply:
Yeah I was thinking this would be simpler since you would not need to divide by ONE_YEAR_IN_MICROS every time